scripts/deploy_erc721Royalties.js

```diff
@@ -1,21 +1,28 @@
 // Right click on the script name and hit "Run" to execute
-const hre = require("hardhat")
+const hre = require('hardhat');
 
 const main = async () => {
-  console.log('Running deployWithEthers script...')
+  console.log('Running deployWithEthers script...');
 
   /* *** ERC721OA *** */
-  const ERC721OA = await hre.ethers.getContractFactory("ERC721Royalties")
-  const erc721OA = await ERC721OA.deploy("RoyaltiesCollection", "RC");
+  const ERC721OA = await hre.ethers.getContractFactory('FOLEARTMX');
```

Comment:
FOLEARTMX no puede ser dinamico o porque esta como estatico el texto?

Reply:
Es el nombre que tenga el contrato al momento de compilar y no se puede cambiar el nombre del contrato al momento de ejecución.

**contracts/ERC721OA/ERC721Royalties.sol: line 8**
`contract FOLEARTMX is ERC721URIStorage, ApprovalsGuardDynamicOwner {`